gaplint.py

```diff
@@ -1427,6 +1433,15 @@ def _parse_cmd_line_args(kwargs) -> Dict[str, Any]:
         help=f"comma separated rule names and/or codes to explain (default: {default})",
     )
 
+    default = _DEFAULT_CONFIG["config-file"]
+    parser.add_argument(
+        "--config-file",
+        nargs="?",
+        type=str,
+        default="",
```

Comment:
```suggestion
        default=None,
```
This fixes the strange error message observed by @stertooy . The issue arose because the empty string is not `None` and so the `__config_yml_path` function returns it as an assumed valid path.